_i18n/fr/resources/user-guides/verification-windows-beginner.md

```diff
@@ -1,254 +1,253 @@
+![certifier pour soi kleo](png/verify_binary_windows_beginner/verify-win_kleopatra-certify-forself.png)
 
-Enter your password, click `OK`.
+Entrez votre mot de passe et cliquer sur `OK`.
 
-![kleo certify pass](png/verify_binary_windows_beginner/verify-win_kleopatra-certify-pinentry.png)
+![certifier mdp kleo](png/verify_binary_windows_beginner/verify-win_kleopatra-certify-pinentry.png)
 
-Click `Finish`.
+Cliquez sur `Terminer`.
 
-![kleo certify finish](png/verify_binary_windows_beginner/verify-win_kleopatra-certify-finish.png)
+![fin certifier kleo](png/verify_binary_windows_beginner/verify-win_kleopatra-certify-finish.png)
 
-## 3. Hash File Verification
+## 3. Vérification du Fichier de Hachage
 
-This section will cover downloading the signed file of known good hashes and verifying its authenticity.
+Cette rubrique couvre le téléchargement du fichier signé des hachages valides et la vérification de son authenticité.
 
-### 3.1. Download Hash File
+### 3.1. Télécharger le Fichier de Hachage
 
-In a web browser, go to the [getmonero.org hash page](https://getmonero.org/downloads/hashes.txt). Right click the page, select `Save Page As`.
+Dans un navigateur internet, rendez-vous sur la [page de haches getmonero.org](https://getmonero.org/fr/downloads/hashes.txt). Faites un clic droit sur la page et sélectionnez `Enregistrer sous...`.
 
-![hashes right click](png/verify_binary_windows_beginner/verify-win_hashes-getmonero-rightclick.png)
+![clic droit hachages](png/verify_binary_windows_beginner/verify-win_hashes-getmonero-rightclick.png)
 
-Leave the default location, click `Save`.
+Laissez l'emplacement par défaut et cliquez sur `Enregistrer`.
 
-![hashes save file](png/verify_binary_windows_beginner/verify-win_hashes-getmonero-savename.png)
+![enregistrer fichier hachages](png/verify_binary_windows_beginner/verify-win_hashes-getmonero-savename.png)
 
-### 3.2. Verify Hash File
+### 3.2. Vérifier le Fichier de Hachage
 
-In Kleopatra, click the `Decrypt/Verify` button.
+Dans Kleopatra cliquez sur le bouton `Déchiffrer / vérifier...`.
 
-![hashes kleo verify button](png/verify_binary_windows_beginner/verify-win_hashes-kleo-verify-button.png)
+![bouton vérifier kleo hachages](png/verify_binary_windows_beginner/verify-win_hashes-kleo-verify-button.png)
 
-Navigate to `Downloads` directory. Select `hashes` file, click `Open`.
+Allez dans le répertoire `téléchargements`. Sélectionnez le fichier `hashes` et cliquez sur `Ouvrir`.
 
-![hashes kleo open file](png/verify_binary_windows_beginner/verify-win_hashes-kleo-verify-button-filename.png)
+![ouvrir fichier kleo hachages](png/verify_binary_windows_beginner/verify-win_hashes-kleo-verify-button-filename.png)
 
-Kleopatra will inform you if the files signature is valid.
+Kleopatra vous indiquera si la signature du fichier est légitime.
 
-If the signature is **VALID** you will see this:
+Si la signature est **LÉGITIME** vous verrez ceci :
```

Comment:
valide or authentique instead of légitime?

Reply:
I found "authentique" more elegant. Should we go with "contrefait" or "falsifié" below?

Reply:
Sure, signature falsifiée sound good.

Reply:
erratum. Kleopatra is saying "Signature valable" or "Signature non valable". so i will stay as it is in the app itself.